examples/mysql-galera/README.md

```diff
@@ -0,0 +1,147 @@
+
+Also, this example requires the image found in the ```image``` directory. For your convenience, it is built and available on Docker's public image repository as ```capttofu/percona_xtradb_cluster_5_6```. It can also be built which would merely require that the image in the pod or replication controller files is updated.
+
+This example was tested on OS X with a Galera cluster running on VMWare using the fine repo developed by Paulo Pires [https://github.com/pires/kubernetes-vagrant-coreos-cluster] and client programs built for OS X.
+
+
+### Running the initial bootstrap node
+
+There is a choice of using a straight pod or a replication controller. There will only ever be one container per pod since this is not an application where adding more translates to any benefit and is actually undesirable.
+
+There are only three pods (hence replication controllers) for this cluster. This number could be increased but would require building on a structure that needs to be inevitably made more dynamic, so for this example, 3 is more than sufficient. The number of nodes must always be odd.
+
+Note: Kubernetes best-practices is to pre-create the services for each controller, and this will work fine for this setup, but a faster initial cluster will result if the services are create after the controller for each single-container pod. The reason for this is that if you provide an IP address in the list of ```sst_cluster_address``` that is not yet ready, it'll take that much longer for the SST election process to complete.
+
+Create this master, using a replication-controller, do as follows:
+
+```kubectl create -f examples/mysql-galera/pxc-node1-controller.yaml```
+
+Next, create a service for the first node:
+
+```kubectl create -f examples/mysql-galera/pxc-node1-service.yaml```
+
+### Running the remaining nodes
+
+Next, repeat the same previous steps for ```pxc-node2``` and ```pxc-node3```
+
+When complete, you should be able connect with a mysql client to the IP address
+of the service for any one of the nodes and find a working cluster
+
+### Confirming replication controller (or pods) and services are running
+
+Using ```kubectl``` from within the ```./examples/mysql-galera``` directory, the status of the lauched replication controllers and services can be confirmed
+
+```
+$ kubectl describe rc pxc-node1
+Name:		pxc-node1
+Namespace:	default
+Image(s):	capttofu/percona_xtradb_cluster_5_6:latest
+Selector:	name=pxc-node1
+Labels:		name=pxc-node1
+Replicas:	1 current / 1 desired
+Pods Status:	1 Running / 0 Waiting / 0 Succeeded / 0 Failed
+No events.
+
+$ kubectl get pods
+NAME              READY     STATUS    RESTARTS   AGE
+kube-dns-sqtap    3/3       Running   0          5d
```

Comment:
(nit) kube-dns is usually run in the kube-system namespace. leave it out here since it's irrelevant. Same below, under services.